### ISSUE TITLE
Error message in output even though Gradle plugin seems to work correctly

### ISSUE THREAD

**Issue by gkapfham** [GatorEducator/gatorgradle#16]
I am running this plugin with the following configuration:

```
---
name: computer-science-111-fall-2017-lab1
break: false
indent: 4
---
src/main:
    java:
        labone/DisplayOutput.java:
            --single 2 --language Java
            --multi 2 --language Java
            --fragment "println(" --count 4
            --fragment "new Date(" --count 1
writing/reflection.md:
    mdl
    --paragraphs 1
    --words 200
``` 

I see this error output:

```
ERROR: GatorGrader management failed, could not checkout 'master' branch!
Command run: 'git' 'checkout' 'master'
OUTPUT: fatal: Not a git repository (or any of the parent directories): .git
Updating GatorGrader...
Already up-to-date.
```

It seems as though the checks are working as I expected. However, I anticipate that this error message may be confusing for the students who read it. Can you please clarify what is happening here, @Michionlion? Is there a way to avoid seeing this error message if everything is, in fact, okay? Or, is there a problem of which I should be aware?

**Comment by gkapfham** [comment]
Hi @Michionlion, I can also confirm that I seem to only see this error when the Gradle plugin is run in a directory that is not a Git repository. If I run this on the sample lab (which is from GitHub) or on a laboratory assignment that is not a Git repository yet, this error will appear. Right now, I am not sure if this is a problem with the plugin or not. But, we should investigate further as time permits because it could be a bug. With that said, I no longer think that it is critical for today's work.

**Comment by Michionlion** [comment]
This issue is actually a bug - the working directory is not set for the command run during dependency management. It is an easy fix however, and should not effect every day use.

**Comment by gkapfham** [comment]
Great, thanks for confirming this issue @Michionlion. I don't think that this is a pressing issue for today. When time permits in the near future, let's work together to fix this bug.

**Comment by gkapfham** [comment]
I think that this bug is resolved, right @Michionlion? Can we close this issue?

**Comment by Michionlion** [comment]
I think that is correct, but I will make sure.

**Comment by gkapfham** [comment]
Hello @Michionlion, can you please confirm that this issue is now resolved? If it is, then I think that it would be a good idea to close this issue, right?

**Comment by Michionlion** [comment]
Yes, it would. The issue was resolved in commit bc2243e4f9716e120a82398fc622bef2a17e914f, by using whatever branch was already checked out, enabling debugging.

**Comment by gkapfham** [comment]
Great, thanks for checking @Michionlion and then closing this issue.
